survey-runner-queue/message_queue.tf

```diff
@@ -14,6 +14,10 @@ resource "aws_instance" "rabbitmq" {
     "${aws_security_group.survey_runner_vpn_sdx_access.id}",
   ]
 
+  root_block_device {
+    delete_on_termination = false
```

Comment:
I think we should make this configurable so we can enable it for developer environments (default to false).